spec/02-integration/05-proxy/15-custom_nginx_directive_spec.lua

```diff
@@ -0,0 +1,58 @@
+local helpers = require "spec.helpers"
+
+
+describe("Custom NGINX directives", function()
+  local proxy_client
+  local bp
+
+  local function start(config)
+    return function()
+      bp.routes:insert {
+        hosts = { "headers-inspect.com" },
```

Comment:
Ah, a super minor thing but which I think we should start paying attention in our tests (perhaps worth an addition to the style guide even?) — I suggest we stop using "valid-looking" URLs in tests, even if we're not hitting them (we've had in the past tests that used an innocent-looking URL which ended up pointing to a porn site!) — we can use instead `example.com` or any URL using the reserved `.test` TLD (which then allows for "descriptive" test hostnames such as `headers-inspect.test`)

Reply:
Thanks for sharing pointing this out!
I like the idea of using the `.test` TLD. I've updated the code to use `headers-inspect.test`.

Reply:
+1

> we've had in the past tests that used an innocent-looking URL which ended up pointing to a porn site

I've also seen cases of test/example code in public documentation using "domain.com", which at one point hosted malware; customers who blindly copy-pasted were in for a nasty surprise :p 

IANA has reserved `example.com` for cases like this, FWIW. https://www.iana.org/domains/reserved

Just my $0.02.